plugins/topology/README.md

```diff
@@ -81,6 +81,57 @@ The Topology plugin enables you to visualize the workloads such as Deployment, J
 
   ```
 
+- The following must be added in`customResources` component in the [`app-config.yaml`](https://backstage.io/docs/features/kubernetes/configuration#configuring-kubernetes-clusters) file to view the edit code decorator clicking on which will take you to the Red Hat OpenShift Dev Spaces instance :
+
+  ```yaml
+   kubernetes:
+     ...
+     customResources:
+        - group: 'org.eclipse.che'
+          apiVersion: 'v2'
+          plural: 'checlusters'
+  ```
+
+  Also, ensure that the CheCluster is granted a [`ClusterRole`](https://backstage.io/docs/features/kubernetes/configuration#role-based-access-control). You can use the following code to grant the `ClusterRole` to the CheCluster :
+
+  ```yaml
+    ...
+    apiVersion: rbac.authorization.k8s.io/v1
+    kind: ClusterRole
+    metadata:
+      name: backstage-read-only
+    rules:
+      ...
+      - apiGroups:
+          - org.eclipse.che
+        resources:
+          - checlusters
+        verbs:
+          - get
+          - list
+
+  ```
+
+- The following annotation(s) are added to workload resources such as Deployments to navigate to the git repo via the edit code decorator.
+
+  ```yaml title="deployment.yaml"
+    ...
+    labels:
+      app.openshift.io/vcs-uri: <GIT_REPO_URL>
```

Comment:
Why do you need an additional url on the Kuberentes resource, if Backstage already provides a standard one for the catalog entry?

https://backstage.io/docs/features/software-catalog/well-known-annotations/#backstageioview-url-backstageioedit-url

I think this is just confusing the user.

Reply:
Yes, I like that idea. I think we should put something along these lines to the readme:

1. The annotations are already populated by the OCP import - so user doesn't have to do anything in that case.
2. In cases when OCP deploy was not via git import, they may need to do it manually and they should not confuse it with `backstage.io/edit-url` annotation since that points to something else and is applied to a completely different entity (backstage catalog entry, not kubernetes resource).

Reply:
@hmanwani-rh Please review the README and lmk if you have any feedback. Thanks!